btx/processing/peak_finder.py

```diff
@@ -15,7 +15,8 @@ class PeakFinder:
     https://confluence.slac.stanford.edu/display/PSDM/Hit+and+Peak+Finding+Algorithms
     """
     
-    def __init__(self, exp, run, det_type, outdir, tag='', mask=None, psana_mask=True, camera_length=None,
+    def __init__(self, exp, run, det_type, outdir, event_receiver=None, event_code=None, event_logic=True,
```

Comment:
Should `event_logic` be set to False by default? I assume that pump-probe experiments are more rare than others?

Reply:
Not essential, but there's an option to run this peak finding from the command line, so we may want to add these as arguments for argparser.

Reply:
Sorry for the confusing name, `event_logic` is used to decide whether we "only" process the given event code, or process "everything but" it.